### PR TITLE
Rewrite some upkeep page postboxes

### PR DESCRIPTION
**Pull request by Moppa5**
#### What are the main changes in this PR?
Rewrite Seravo Plugin update and Change PHP version postboxes. Also, add radio button to templates and rewrite PHP compatibility check to use the new AJAX poller.

Also, removed some excessive AJAX and JS files that could have caused issues with the new design. These files should be fully removed when the rewrite is complete on the given page.

### CODE REVIEW

**Comment by JoosuaKoskinen** [comment]
> remove the old check PHP compatibility as it needs to be re-written so that it doesn't timeout
It could be rewritten already as we have to poller merged.

File: src/modules/upkeep.php [hunk 1]
```diff
@@ -63,33 +67,50 @@ public static function load() {
...
-     */
     public static function init_upkeep_postboxes( Toolpage $page ) {
+      /**
+       * Seravo Plugin Updater
+       */
+      $seravo_plugin_update = new Postbox\Postbox('seravo-plugin-updater');
+      $seravo_plugin_update->set_title(__('Seravo Plugin Updater', 'seravo'));
+      $seravo_plugin_update->set_requirements(array( Requirements::CAN_BE_ANY_ENV => true ));
+
+      $update_button = new SimpleCommand('seravo-plugin-update', 'wp-seravo-plugin-update &');
```
Comment (JoosuaKoskinen) [inline src/modules/upkeep.php:80]: The other postboxes and AJAX handlers use namespace prefix, this one should too.
`Ajax\SimpleCommand`

File: src/modules/upkeep.php [hunk 2]
```diff
@@ -187,7 +208,196 @@ public static function register_scripts( $page ) {
...
+
+    /**
+     * Build Seravo Plugin Update postbox.
+     * @param Component $base Postbox base component.
+     * @param Postbox $postbox Current postbox to build for.
+     * @param mixed $data Data returned by data function.
+     */
+    public static function build_seravo_plugin_update_postbox( Component $base, Postbox\Postbox $postbox, $data ) {
+      if ( ! isset($data['current_version']) || ! isset($data['upstream_version']) ) {
+        $base->add_child(Template::error_paragraph(__('No upstream or current Seravo Plugin version available, please try again later')));
```
Comment (JoosuaKoskinen) [inline src/modules/upkeep.php:369]: Translate missing domain. Add `'seravo'`.

File: src/modules/upkeep.php [hunk 3]
```diff
@@ -187,7 +208,196 @@ public static function register_scripts( $page ) {
 
         wp_localize_script('seravo_upkeep', 'seravo_upkeep_loc', $loc_translation);
       }
+    }
+
+    /**
+     * Build Change PHP version postbox.
+     * @param Component $base Postbox base component.
+     * @param Postbox $postbox Current postbox to build for.
+     * @param mixed $data Data returned by data function.
```
Comment (JoosuaKoskinen) [inline src/modules/upkeep.php:217]: You are not using $data param

File: src/modules/upkeep.php [hunk 4]
```diff
@@ -187,7 +208,196 @@ public static function register_scripts( $page ) {
...
+
+      if ( $polling === false ) {
+        if ( array_key_exists($php_version, $php_version_array) ) {
+
+          if ( $php_version === $current_version ) {
+            $response = new AjaxResponse();
+            $response->is_success(true);
+            $response->set_data(
+              array(
+                'output' => Template::error_paragraph(__('The selected PHP version is already in use.', 'seravo'))->to_html(),
```
Comment (JoosuaKoskinen) [inline src/modules/upkeep.php:331]: Another `Template->to_html`

File: src/modules/upkeep.php (hunk 1 above)
Comment (JoosuaKoskinen) [inline src/modules/upkeep.php:80]: `wp-seravo-plugin-update` is just ran on background? What happens after the update is done? Is anything outputted to user? I'm not completely sure how this is supposed to work and this is bit hard to test.

**Review by JoosuaKoskinen — CHANGES_REQUESTED** [review]
Also, none of the old postboxes worked anymore because JavaScript seems to be broken.

**Comment by JoosuaKoskinen** [comment]
Was the Javascript fixed yet? Still couldn't get it to work.

**Comment by Moppa5** [comment]
@JoosuaKoskinen Should be fixed by now. At least I could get the old postboxes working. Are you sure you have pulled the new work?

**Review by JoosuaKoskinen — APPROVED** [review]

**Pull request merged**